Bottom: d1639c4eab0317eff78ce528910f9f1265c098b9
Top:    d6ef2da464caa0c937e4ca2fb4e7b1918d0d59f9
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2016-01-07 01:00:08 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
